src/cosalib/cmdlib.py

```diff
@@ -248,6 +247,18 @@ def image_info(image):
             ['qemu-img', 'info', '--output=json', image],
             capture_output=True).stdout
         )
+
+        # Fixed VPC/VHD v1 disks are really raw images with a VHD footer.
+        # The VHD footer uses 'conectix' as the identify in first 8 bytes
+        # of the last 512 bytes. Sadly, 'qemu-img' does not identify it
+        # properly.
+        if out.get("format") == "raw":
+            with open(image, 'rb') as imgf:
+                imgf.seek(-512, os.SEEK_END)
+                data = (imgf.read(8)).decode()
```

Comment:
This will throw an exception if the data isn't valid `UTF-8` right?  Why not just keep it as bytes and do:

```
if data == b"connectix":
```

?

Reply:
Duh...I was trying to be too clever. Fixed and repushed.